SimPPS/Configuration/python/PPSTransport_cff.py

```diff
@@ -0,0 +1,20 @@
+import FWCore.ParameterSet.Config as cms
+from SimG4Core.Application.g4SimHits_cfi import *
+g4SimHits.Generator.MinEtaCut = cms.double(-13.0)
+g4SimHits.Generator.MaxEtaCut = cms.double( 13.0)
+g4SimHits.Generator.HepMCProductLabel   = 'LHCTransport'
+g4SimHits.SteppingAction.MaxTrackTime = cms.double(2000.0)
+g4SimHits.StackingAction.MaxTrackTime = cms.double(2000.0)
```

Comment:
Configuration fragments modifying objects defined in other files is in generally discouraged (even though we still have many files that do it). Could these be set in `g4SimHits_cfi` e.g. with the `ctpps_2016` modifier (that appears to be a general switch to turn on PPS stuff)?

Reply:
@makortel What do you propose? These parameters are defined in SimG4Core/Applications/python/g4SimHits_cfi.py, but I'm afraid to cause undesired impact somewhere else...

Reply:
Yes, we need this for all years since 2016, otherwise, PPS tracks will not be treated properly. Should I include g4SimHits_cfi in this PR then?

Reply:
> Should I include g4SimHits_cfi in this PR then?

Yes please.

Reply:
I noticed there is a modified "run2_common", since PPS is intended to be included in all run2, isn't a better idea to use this modified instead of the ctpps_[year]?

Reply:
The convention has been to use the "most specialized" modifier for the modifications. E.g. in this case that allows to easily construct a workflow like "Run 2 without PPS". (also strictly speaking Run 2 started in 2015, while the first `ctpps` Modifier is for 2016.)

Reply:
Ok.

Reply:
Done. It was used only ctpps_2016 (and tested against 2017 and 2018 as well) as this modified is a general one for PPS.